main.go

```diff
@@ -208,11 +212,40 @@ func run(state overseer.State) {
 		handlers.SetArchiveMaxTimeout(*archiveTimeout)
 	}
 
+	// Build include and exclude detector filter sets.
+	var includeDetectorTypes, excludeDetectorTypes map[detectorspb.DetectorType]struct{}
+	{
+		includeList, err := config.ParseDetectors(*includeDetectors)
+		if err != nil {
+			// Exit if there was an error to inform the user of the misconfiguration.
+			logger.Error(err, "invalid detector configuration")
+			os.Exit(1)
+		}
+		excludeList, err := config.ParseDetectors(*excludeDetectors)
+		if err != nil {
+			// Exit if there was an error to inform the user of the misconfiguration.
+			logger.Error(err, "invalid detector configuration")
+			os.Exit(1)
+		}
+		includeDetectorTypes = detectorTypeToSet(includeList)
+		excludeDetectorTypes = detectorTypeToSet(excludeList)
+	}
+	includeFilter := func(d detectors.Detector) bool {
+		_, ok := includeDetectorTypes[d.Type()]
+		return ok
+	}
+	excludeFilter := func(d detectors.Detector) bool {
+		_, ok := excludeDetectorTypes[d.Type()]
+		return !ok
+	}
```

Comment:
__optional suggestion__: It might be good to determine if there is any overlap in the exclude vs include and potentially error out, as what should happen in that case is somewhat unknown (though from what I can tell, overlaps would be excluded as excluded filters are applied last).

Reply:
This is a good call out, but I think that might be more common than not with the way I constructed this.

By default the include list is `all` so any detector in the exclude list will trigger this case. It's subtractive by nature (we start with all possible detectors), then we filter out any detectors not in the include list, then we filter out any detectors in the exclude list.

Reply:
I've updated the `--exclude-detectors` help text to explain it takes precedence.